cmd/zrepl/zrepl.c

```diff
@@ -237,7 +248,10 @@ uzfs_zvol_io_receiver(void *arg)
 		(void) pthread_mutex_lock(&zinfo->zinfo_mutex);
 	}
 	(void) pthread_mutex_unlock(&zinfo->zinfo_mutex);
-	uzfs_zinfo_drop_refcnt(zinfo, B_FALSE);
+
+	close(fd);
+	LOG_INFO("Data connection closed");
```

Comment:
we should remove this message because in ack thread we already log similar message and seeing this message twice in log is confusing.

Reply:
Done.